setup.py

```diff
@@ -26,6 +26,9 @@ def prepend_find_packages(*roots):
         
     return packages
 
+if os.environ.get('CONDA_DLL_SEARCH_MODIFICATION_ENABLE', '0') == '0':
```

Comment:
Looks good! Please add a note explaining why we need to add this and maybe referencing that github thread